silk-core/src/main/scala/org/silkframework/runtime/plugin/PluginContext.scala

```diff
@@ -23,39 +23,44 @@ trait PluginContext {
 
   def templateVariables: TemplateVariablesReader
 
+  def workflowId: Option[Identifier]
```

Comment:
Please add a comment, it is not clear what role a workflow plays in context of plugin creation and update.

Reply:
I have added comments to the attributes. Also made it clear that PluginContext is also used for plugin execution.